### PR TITLE
Adding Pegasus landing page tests to 8 more of our top 15 pages

### PR DESCRIPTION
**Pull request by hannahbergam**
@dmcavoy  pointed out that we only have current UI test coverage for some of our top visited Pegasus pages. This PR adds coverage for many of the missing pages, through to the fifteenth spot. I kept these across different files to prevent flakiness, as the original file was a combination of files 1 and _2 and had just enough urls to have an unfortunate flakiness report.

Screenshot 2024-06-25 at 4 37 41 PM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/test/ui/features/hour_of_code/tutorial_landing_pages_3.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Scenario Outline: Simple page view
+  When I open my eyes to test "<test_name>"
+  And I am on "<url>"
+  And I dismiss the language selector
+  And I wait for the video thumbnails to load
+  Then I see no difference for "initial load"
+  And I close my eyes
+  And I sign out
+Examples:
+  | url                                                               | test_name                  |
```
Comment (bethanyaconnor) [inline dashboard/test/ui/features/hour_of_code/tutorial_landing_pages_3.feature]: Are these the headers of a table? Or leftover from something?

File: dashboard/test/ui/features/hour_of_code/tutorial_landing_pages_3.feature
Comment (bethanyaconnor) [inline dashboard/test/ui/features/hour_of_code/tutorial_landing_pages_3.feature:1]: I know that the previous two files are in the `/hour_of_code/` directory, but that doesn't seem like the best place for these. Should we consider moving all 4 files into the `/acquisition_products/` directory?

File: dashboard/test/ui/features/hour_of_code/tutorial_landing_pages_3.feature (hunk 1 above)
Comment (hannahbergam) [reply dashboard/test/ui/features/hour_of_code/tutorial_landing_pages_3.feature]: These are the headers of the table columns that are used in the lines above to insert the name and url of each line:)

File: dashboard/test/ui/features/hour_of_code/tutorial_landing_pages_3.feature
Comment (hannahbergam) [reply dashboard/test/ui/features/hour_of_code/tutorial_landing_pages_3.feature:1]: I moved them all into our directory and renamed them- let me know if you prefer a different naming convention for them!

Comment (bethanyaconnor) [reply dashboard/test/ui/features/hour_of_code/tutorial_landing_pages_3.feature:1]: Yay looks good! Thank you for doing this!

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**